bindings_ffi/src/mls.rs

```diff
@@ -7960,4 +7960,50 @@ mod tests {
             FfiConsentState::Denied
         );
     }
+
+    #[tokio::test]
+    async fn test_sync_messages() {
+        let alix = Tester::builder()
+            .with_sync_server()
+            .with_sync_worker()
+            .build()
+            .await;
+        let bo = Tester::new().await;
+
+        let alix_group = alix
+            .conversations()
+            .create_group_with_inbox_ids(vec![bo.inbox_id()], FfiCreateGroupOptions::default())
+            .await
+            .unwrap();
+
+        alix_group.send_text("hi").await.unwrap();
+
+        let initial_messages = alix_group.find_messages(FfiListMessagesOptions::default()).await.unwrap();
+        assert_eq!(initial_messages.len(), 2);
+
+        let alix2 = alix.builder.build().await;
+
+        let state = alix2.inbox_state(true).await.unwrap();
+        assert_eq!(state.installations.len(), 2);
+        
+        // Doesn't fail on the not finding the group if you send a message after the new installation is created.
+        // alix_group.send_text("hi").await.unwrap();
```

Comment:
This piece doesn't make sense to me. Seems like sending a message in a group will automatically add the installation? But syncing doesn't seem to be enough?